### PR TITLE
[REVIEW] dask infra updates

### PR DESCRIPTION
**Pull request by Iroy30**
Small fixes for when https://github.com/rapidsai/raft/pull/7 merged

### CODE REVIEW

File: conda/environments/cugraph_dev_cuda10.0.yml [hunk 1]
```diff
@@ -14,10 +14,10 @@ dependencies:
 - dask-cuda=0.14*
 - dask-cudf=0.14*
 - nccl>=2.5
+- ucx-py=0.14*
```
Comment (raydouglass) [inline conda/environments/cugraph_dev_cuda10.0.yml:17]: Is this an actual change in dependencies for cugraph? If so, the conda recipes need to be updated as well.

Comment (afender) [reply conda/environments/cugraph_dev_cuda10.0.yml:17]: Yes, and probably `ci/gpu/build.sh` around line 70 too?

Comment (Iroy30) [reply conda/environments/cugraph_dev_cuda10.0.yml:17]: Thanks, I have added those

**Comment by afender** [comment]
rerun tests

File: conda/recipes/cugraph/meta.yaml [hunk 2]
```diff
@@ -28,11 +28,13 @@ requirements:
     - cython>=0.29,<0.30
     - libcugraph={{ version }}
     - cudf={{ minor_version }}
+    - ucx-py {{ minor_version }}
   run:
     - python x.x
     - libcugraph={{ version }}
     - cudf={{ minor_version }}
     - nccl>=2.5
```
Comment (raydouglass) [inline conda/recipes/cugraph/meta.yaml:36]: You removed nccl from the environment files. Should it also be removed here?

File: ci/gpu/build.sh [hunk 3]
```diff
@@ -68,6 +68,7 @@ conda install -c nvidia -c rapidsai -c rapidsai-nightly -c conda-forge -c defaul
       dask-cuda=${MINOR_VERSION} \
       scikit-learn>=0.21 \
       nccl>=2.5 \
```
Comment (raydouglass) [inline ci/gpu/build.sh:70]: You removed nccl from the environment files. Should it also be removed here?

Comment (Iroy30) [reply ci/gpu/build.sh:70]: nccl is still needed in the environment. I removed nccl as it was redundant. We already have a nccl>=2.5 dependency

File: conda/environments/cugraph_dev_cuda10.1.yml [hunk 4]
```diff
@@ -13,10 +13,10 @@ dependencies:
 - dask-cuda=0.14*
 - dask-cudf=0.14*
 - nccl>=2.5
+- ucx-py=0.14*
 - scipy
 - networkx
 - python-louvain
-- nccl
```
Comment (Iroy30) [inline conda/environments/cugraph_dev_cuda10.1.yml:19]: removed nccl as there is already a dependency

**Pull request merged**